### ISSUE TITLE
Facing issue when displaying two different pdf side by side on same screen ( Android )

### ISSUE THREAD

**Issue by Sharvin26** [xcarpentier/rn-pdf-reader-js#114]
#### Issue Description

I am seeing a weird behavior when displaying the PDF side by side on same screen. I am using Flatlist to show the PDF Documents side by side.

I am seeing same PDF document side by side when both PDF that I have uploaded are different. On IOS I am able to see different PDF side by side but this issue only occurs on Android. Please see the screenshot below to see the issue and behavioral difference between IOS and android.

**Android:**
![Screenshot_20200710-121517](https://user-images.githubusercontent.com/17084209/87133940-a61cdd00-c2b5-11ea-831e-d0918836245c.jpg)

**IOS:**
![Simulator Screen Shot - iPhone 11 - 2020-07-10 at 12 19 00](https://user-images.githubusercontent.com/17084209/87133971-afa64500-c2b5-11ea-95e5-09faf6efce82.png)

#### Steps to Reproduce / Code Snippets

Steps are mentioned in this [issue](https://github.com/xcarpentier/rn-pdf-reader-js/issues/110#issue-648708778).

#### Expected Results
It should show different PDF Documents side by side on Android. ( As it shows on IOS )

#### Additional Information

* Nodejs version: v12.16.1
* React version: 16.9.0
* React Native version: 0.61
* Expo version: 37.0.1
* rn-pdf-reader-js version: 4.0
* Platform(s) (iOS, Android, or both?): Android
* TypeScript version: Not Using

**Comment by xcarpentier** [comment]
Actually we are not supported this kind of so called “multiple pdf render at the same time” yet!

But it seems a recurrent feature request.

If some others have this kind of demand, feels free to thumbs up and I will try to make it possible.

**Comment by Sharvin26** [comment]
Yes, it would be great add on if the library could render multiple PDF at a time. Also why is the PDF rendering slow for android but fast for the IOS device? It would be great if you could comment out the log statement such as `URL_TO_BASE64`.
